src/Symfony/Component/Mailer/Transport/NativeTransportFactory.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Transport;
+
+use Symfony\Component\Mailer\Exception\TransportException;
+use Symfony\Component\Mailer\Exception\UnsupportedSchemeException;
+use Symfony\Component\Mailer\Transport\Smtp\SmtpTransport;
+use Symfony\Component\Mailer\Transport\Smtp\Stream\SocketStream;
+
+/**
+ * @author Laurent VOULLEMIER <laurent.voullemier@gmail.com>
```

Comment:
I think we need to add a small description about this transport as `Native` is not really descriptive. I would also copy/paste the recos about which args to pass to sendmail (like in the sendmail transport). And probably add a note in the Sendmail transport about this transport and the main differences?

Do we want to deprecated the sendmail transport then?

Reply:
Some documentation added. Not sure about deprecating the sendmail transport factory, I tend to think sendmail on php.ini may be misconfigured. In this case rely on standard sendmail path and recommended options seems to me a better way.